RELEASE_NOTES.md

```diff
@@ -168,7 +168,7 @@ Chef now ships with Ruby 2.6.3. This new version of Ruby improves performance an
 
 Platform and platform_family detection on Linux has been rewritten to utilize the latest config files on modern Linux distributions before falling back to slower and fragile legacy detection methods. Ohai will now begin by parsing the contents of `/etc/os-release` for OS information if available. This feature improves the reliability of detection on modern distros and allows detection of new distros as they are released.
```

Comment:
Did 'Ohai' get a name change with all the other product name changes?

Reply:
Ohai did not a rename